google/cloud/forseti/notifier/notifier.py

```diff
@@ -67,8 +67,8 @@ def convert_to_timestamp(violations):
             created_at_datetime converted to timestamp string.
     """
     for violation in violations:
-        violation.created_at_datetime = (
-            violation.created_at_datetime.strftime(
+        violation['created_at_datetime'] = (
```

Comment:
The Args/Returns docs need to be updated since this not a sqlalchemy_object anymore.

Reply:
Done.